README.md

```diff
@@ -246,7 +247,7 @@ See [CONTRIBUTING.md](./docs/CONTRIBUTING.md).
   [-d <RTL-SDR USB device index>] (default: 0)
   [-d :<RTL-SDR USB device serial (can be set with rtl_eeprom -s)>]
 	To set gain for RTL-SDR use -g <gain> to set an overall gain in dB.
-	SoapySDR device driver is available.
+	SoapySDR device driver is not available.
```

Comment:
@zuckschwerdt it's unfortunate that the `maintainer_update.py` script creates these diffs. Should we do something about that? It think filtering this information from the man page / README is fine, as it depends on the build anyways.

Reply:
it's just a lazy default because we really needed some up-to-date text. Any changes or improved workflow welcome. (would be great to also have the rtl_433.example.conf protocols in sync)